### PR TITLE
Math folder

### PR DESCRIPTION
**Pull request by Ved-Mahajan**
This is to start the math folder and have collatz sequence as the starter!

### CODE REVIEW

File: math/collatz_sequence.jl [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+function main()
+	n = 42
+	seq = collatz_sequence(n)
+	println(seq)
+	println("It took $(length(seq)) steps for n = $(n)")
+end
+
+main()
+
```
Comment (Panquesito7) [inline math/collatz_sequence.jl:28]:
```suggestion
```

**Review by Panquesito7 — DISMISSED** [review]
I don't really know Julia, but the program itself seems to be working fine. Thanks!

**Comment by Ved-Mahajan** [comment]
I am sorry for this closing and opening. I take full responsibilities for the inconvenience.

**Comment by maltejur** [comment]
On some files you used unit tests. Should we make this a requirement for all algorithms?

**Comment by Ved-Mahajan** [comment]
> On some files you used unit tests. Should we make this a requirement for all algorithms?

I think that this is a good idea as it is a good way for someone using the code to know that the code works with those tests.
It is just like `doctest` in python.

**Comment by maltejur** [comment]
Nice. Can you add tests to all your files like you did in `math/line_length.jl` (with the `@testset`)?

**Comment by Panquesito7** [comment]
> On some files, you used unit tests. Should we make this a requirement for all algorithms?

Sounds good to me. We can add it to the contributing guidelines. 👍

**Comment by Ved-Mahajan** [comment]
> Nice. Can you add tests to all your files like you did in `math/line_length.jl` (with the `@testset`)?

I did that :-)

**Review by Panquesito7 — APPROVED** [review]

**Comment by maltejur** [comment]
Just some small changes https://github.com/Ved-Mahajan/Julia/pull/2

**Review by Panquesito7 — APPROVED** [review]

**Review by maltejur — APPROVED** [review]
Everything is looking good to me. Thanks for your contribution!

**Pull request merged**